roles/openshift_aws/tasks/dns_route53_record.yml

```diff
@@ -0,0 +1,22 @@
+---
+- debug: msg="{{ openshift_aws_dns_records[l_openshift_aws_dns_element] }}"
+
+- name: querying elb
+  ec2_elb_facts:
+    names: "{{ openshift_aws_dns_records[l_openshift_aws_dns_element]['value'] }}"
+    region: "{{ openshift_aws_region }}"
+  register: elb_facts
+  when:
+    - "'elb' in openshift_aws_dns_records[l_openshift_aws_dns_element]['kind']"
+    - "'CNAME' in openshift_aws_dns_records[l_openshift_aws_dns_element]['type']"
+
+- name: creating record
+  route53:
+    command: create
+    overwrite: no
+    private_zone: "{{ (openshift_aws_dns_records[l_openshift_aws_dns_element]['private_zone'] == False) | ternary('no', 'yes') }}"
```

Comment:
No need for ternary here, just cast to boolean.  yes == True and no == False in yaml.